src/analysis/processing/qgsalgorithmlayoutatlastoimage.cpp

```diff
@@ -0,0 +1,250 @@
+/***************************************************************************
+                         qgsalgorithmlayoutatlastoimage.cpp
+                         ---------------------
+    begin                : June 2020
+    copyright            : (C) 2020 by Mathieu Pellerin
+    email                : nirvn dot asia at gmail dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "qgsalgorithmlayoutatlastoimage.h"
+#include "qgslayout.h"
+#include "qgslayoutatlas.h"
+#include "qgsprintlayout.h"
+#include "qgsprocessingoutputs.h"
+#include "qgslayoutexporter.h"
+
+#include <QImageWriter>
+
+///@cond PRIVATE
+
+QString QgsLayoutAtlasToImageAlgorithm::name() const
+{
+  return QStringLiteral( "atlaslayouttoimage" );
+}
+
+QString QgsLayoutAtlasToImageAlgorithm::displayName() const
+{
+  return QObject::tr( "Export atlas layout as image" );
+}
+
+QStringList QgsLayoutAtlasToImageAlgorithm::tags() const
+{
+  return QObject::tr( "layout,atlas,composer,composition,save,png,jpeg,jpg" ).split( ',' );
+}
+
+QString QgsLayoutAtlasToImageAlgorithm::group() const
+{
+  return QObject::tr( "Cartography" );
+}
+
+QString QgsLayoutAtlasToImageAlgorithm::groupId() const
+{
+  return QStringLiteral( "cartography" );
+}
+
+QString QgsLayoutAtlasToImageAlgorithm::shortDescription() const
+{
+  return QObject::tr( "Exports an atlas layout atlas as image." );
+}
+
+QString QgsLayoutAtlasToImageAlgorithm::shortHelpString() const
+{
+  return QObject::tr( "This algorithm outputs an atlas layout image file (e.g. PNG or JPEG images)." );
```

Comment:
```suggestion
  return QObject::tr( "This algorithm outputs an atlas layout to a set of image files (e.g. PNG or JPEG images)." );
```

I'd also suggest expanding a little here an explaining that the coverage layer and other atlas parameters can be overridden.